docs/manual_doc/Statement_Syntax/Utility_Statements/USE_syntax.md

```diff
@@ -8,12 +8,12 @@ The named space remains the default until the end of the session or another USE
 
 ```
 USE space1
-GO FROM 1 OVER edge1   # Traverse in graph space1
+GO FROM 1 OVER edge1   -- Traverse in graph space1
 USE space2
-GO FROM 1 OVER edge1   # Traverse in graph space2
-USE space1;            # switch to space1
+GO FROM 1 OVER edge1   -- Traverse in graph space2. These nodes and edges have no relevant with space1.
+USE space1;            -- Not you are back to space1. Hereafter, you can not operate (CRUD) any data in space2.
```

Comment:
Now...

operate data is kind of strange
operate>- query
or
you can not make any data changes in space2